service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/azure/container/VerifyContainerResourceDoesNotExist.java

```diff
@@ -0,0 +1,53 @@
+package bio.terra.workspace.service.resource.controlled.flight.clone.azure.container;
+
+import bio.terra.common.exception.ValidationException;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import bio.terra.stairway.exception.RetryException;
+import bio.terra.workspace.common.utils.FlightUtils;
+import bio.terra.workspace.db.ResourceDao;
+import bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class VerifyContainerResourceDoesNotExist implements Step {
```

Comment:
I don't see any direct test coverage for this. Probably worth testing that it does detect if a container already exists.

Reply:
Sure, good call: https://github.com/DataBiosphere/terra-workspace-manager/pull/883/commits/49ad9f3e1092c778b585ec6f03aa19565a6ac417